### PR TITLE
Add gRPC API examples in JS. Add TLS notes and support email

### PR DESCRIPTION
**Pull request by alexromanov**

### CODE REVIEW

File: examples/js/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+1. Make sure, that you have registered an account in [PSG Self Serve UI](https://psg.iog.services/), purchased a package and generated API Token.
+2. Set AWS S3 connection details in src/store_and_hash_client.js:
+```shell
+const reqs = [
+    new StoreAndHashMessages.StoreAndHashRequest()
+        .setDetails(
+            new StoreAndHashMessages.UploadDetails()
+                .setAws(new StoreAndHashMessages.AwsCredentials()
+                    .setKeyid("AWS_ACCCESS_KEY")
```
Comment (mcsherrylabs) [inline examples/js/README.md:48]: I think we need a comment here about either replacing these with actual values or that they are loading from the env... it's not clear to me.

File: examples/js/src/store_and_hash_client.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    'psg-testnet.iog.services:2001',
+    grpc.credentials.createSsl()
+);
+
+const reqs = [
+    new StoreAndHashMessages.StoreAndHashRequest()
+        .setDetails(
+            new StoreAndHashMessages.UploadDetails()
+                .setAws(new StoreAndHashMessages.AwsCredentials()
+                    .setKeyid("AWS_ACCCESS_KEY")
```
Comment (mcsherrylabs) [inline examples/js/src/store_and_hash_client.js:17]: Same comment as above ... I'm just not sure if I should replace these values or how they get into the code....

**Review by mcsherrylabs — CHANGES_REQUESTED** [review]
2 tiny changes please. But this is great. Very helpful.

File: examples/js/README.md (hunk 1 above)
Comment (alexromanov) [reply examples/js/README.md:48]: Updated

File: examples/js/src/store_and_hash_client.js (hunk 2 above)
Comment (alexromanov) [reply examples/js/src/store_and_hash_client.js:18]: Fixed too.

**Review by mcsherrylabs — APPROVED** [review]

**Pull request merged**